model/registry/instrumentation.yaml

```diff
@@ -0,0 +1,24 @@
+groups:
+- id: registry.instrumentation
+  prefix: instrumentation
+  type: attribute_group
+  brief: >
+    Describes properties about the library that is being instrumented or that
+    is responsible for performing the instrumentation.
+  attributes:
+    - id: source.repo
```

Comment:
npm, python, maven, nuget allow a general-purpose link to anything and call it:
- `homepage` - npm
- `project_urls` - pypi
- `url` - maven
- `projectUrl` - nuget

This 1) describes the format (URL) 2) allows any link to be added, not just the library repo which can be closed source, etc.

So I'd prefer to call this attribute something like `instrumentation.scope|library.project_url`

Reply:
My biggest concern with this suggestion is that it seems to be language-specific, whereas the aim of that work was towards facilitating cross-language analysis of cross-platform / multi-platform libraries that have different per-language variants. Regardless, this is no longer in the scope of this PR. Let's continue that discussion in https://github.com/open-telemetry/semantic-conventions/pull/1094